### PR TITLE
Add favicon

### PR DESCRIPTION
**Pull request by lafkpages**
Add the [Replit logo](https://replit.com/public/icons/favicon-prompt-192.png) as favicon.

### CODE REVIEW

File: views/_layout.ejs [hunk 1]
```diff
@@ -18,6 +18,9 @@
 			type="text/css" />
 		<!--  -->
 		<link href="/style.css" rel="stylesheet" type="text/css" />
+		
+		<!-- Favicon -->
+		<link rel="icon" href="https://replit.com/public/icons/favicon-prompt-192.png" type="image/png" sizes="192x192">
```
Comment (PotentialStyx) [inline views/_layout.ejs:23]: Currently `views/service/` and `views/middleware/` both use their own _layout.ejs, could you add this change to those files as well

**Review by PotentialStyx — DISMISSED** [review]

File: views/_layout.ejs (hunk 1 above)
Comment (lafkpages) [reply views/_layout.ejs:23]: Ok

Comment (lafkpages) [reply views/_layout.ejs:23]: Done

**Review by PotentialStyx — APPROVED** [review]
LGTM 👍

**Comment by lafkpages** [comment]
Favicon isn't showing up on the site. Is it like deployments where you have to redeploy or smth?

**Comment by PotentialStyx** [comment]
I currently have to manually redeploy, I'll do that later.

**Pull request merged**